client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -297,9 +298,36 @@ const Editor = (props: EditorProps): JSX.Element => {
     // TODO this should *probably* be set on focus
     editorRef.current = editor;
     data.editor = editor;
+
+    const storedAccessibilityModes = () => {
+      const accessibilityMode = {
+        isAccessibilityModeOn: props.inAccessibilityMode
+      };
+
+      type AccessibilityMode = typeof accessibilityMode;
+
+      const accessibility = store.get('accessibilityMode') as AccessibilityMode;
+      if (!accessibility) {
+        store.set('accessibilityMode', accessibilityMode);
+      }
+      // Only able to set the arialabel when it is set to true
+      // Otherwise it gets overwritten by the monaco default aria-label
+      if (accessibility?.isAccessibilityModeOn) {
+        editor.updateOptions({
+          ariaLabel:
+            'Accessibility mode set to true press ctrl + e to disable; Press alt F1 for more options'
```

Comment:
```suggestion
            'Accessibility mode set to true. Press ctrl + e to disable;'
```
Isn't `alt F1` system specific?  On Ubuntu that shows all windows at once.  `F1` brings up the list of commands on my system, but I don't know if that's universal.

Reply:
The default message when accessibility mode is disabled is:

"The editor is not accessible at this time. Press Alt+F1 for options."

Just to be sure can you take a look at your Ubuntu system if it is the same?

To get there easily:
1. Open dev tools in Chrome
2. If you are not already go to the elements page
3. Press CTRL + F and search for textarea

Reply:
```suggestion
            'Accessibility mode set to true. Press Ctrl+e to disable or press Alt+F1 for more options'
```
That's unfortunate, but I guess it would be more confusing if we did something custom.  Thanks for explaining, I've updated the suggestion to just tweak the wording.